cli/src/domains/spree/getSpree.ts

```diff
@@ -1,56 +1,48 @@
-import inquirer from 'inquirer';;
-import type Integration from "../integration/Integration";
+import inquirer from 'inquirer';
+import type Spree from './Spree';
 
 type Answers = {
-  integration: Integration;
+  spree: Spree;
 };
 
-/** The answers expected in the form of 'inquirer'. */
-
 type Options = {
   message: string;
 };
 
-const integrations = [
+const spree = [
```

Comment:
Could it be changed to a more meaningful name? **Spree** suggests the particular instance, not an array of installation versions.

```suggestion
const spreeVariants = [
```